cranelift/filetests/filetests/alias/simple-alias.clif

```diff
@@ -0,0 +1,92 @@
+test compile precise-output
+set opt_level=speed
+target aarch64
```

Comment:
I think these tests would be a lot easier to read as CLIF to CLIF tests. That way we don't have to figure out what changes are from unrelated passes or just related to aarch64-specific lowering, or even have to know aarch64 to understand these tests that are logically portable and architecture-independent.

Adding `test alias_analysis` that just does alias analysis followed by GVN by calling `Context::remove_redundant_loads` should be really straightforward. Should be able to pretty much copy+paste the `test simple_gvn` implementation:

https://github.com/bytecodealliance/wasmtime/blob/0a0c232a14e651f1457ced0ad0c950771de2f3c3/cranelift/filetests/src/test_simple_gvn.rs

Reply:
That's a great idea, thanks -- done!